### PR TITLE
Added option to reverse the order of years.

### PR DESCRIPTION
**Pull request by ferdyrod**
Hi,

I added an option to the picker Builder to display the years in reverse order. 

If you want to include it.

Regards

### CODE REVIEW

File: monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerDialog.java [hunk 1]
```diff
@@ -179,6 +179,8 @@ private void showYearOnly() {
         _monthPicker.showYearOnly();
     }
 
+    private void setReverseYearOrder(Boolean reverseOrder) { _monthPicker.setReverseYearOrder(reverseOrder); }
```
Comment (SufianBabri) [inline monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerDialog.java]: Primitive boolean can be used instead.
https://stackoverflow.com/questions/3728616/boolean-vs-boolean-in-java

File: monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerDialog.java [hunk 2]
```diff
@@ -479,6 +482,16 @@ public Builder setOnYearChangedListener(OnYearChangedListener onYearChangedListe
             return this;
         }
 
+        /**
+         * User can reverse the order of the years to display. Default is false.
+         *
+         * @return Builder
+         */
+        public Builder setReverseYearOrder(Boolean reverseOrder) {
```
Comment (SufianBabri) [inline monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerDialog.java]: same here

File: monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerView.java [hunk 3]
```diff
@@ -406,6 +406,10 @@ protected void setOnYearChangedListener(MonthPickerDialog.OnYearChangedListener
         }
     }
 
+    protected void setReverseYearOrder(Boolean reverseOrder) {
```
Comment (SufianBabri) [inline monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerView.java]: same here

File: monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/YearPickerView.java [hunk 4]
```diff
@@ -94,6 +94,10 @@ public void setColors(HashMap<String, Integer> colors) {
         this._colors = colors;
     }
 
+    public void setReverseYearOrder(Boolean reverseOrder) {
```
Comment (SufianBabri) [inline monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/YearPickerView.java]: same here

**Comment by achatina** [comment]
Thanks! Very useful thing! Waiting for see it in library

File: monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerDialog.java (hunk 1 above)
Comment (ferdyrod) [reply monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerDialog.java]: done

**Comment by ferdyrod** [comment]
done

**Pull request closed without merging**